crates/context/primitives/src/client.rs

```diff
@@ -281,4 +283,42 @@ impl ContextClient {
 
         receiver.await.expect("Mailbox not to be dropped")
     }
+
+    pub async fn grant_permission(
+        &self,
+        context_id: ContextId,
+        granter_id: PublicKey,
+        grantee_id: PublicKey,
+        capability: Capability,
+    ) -> eyre::Result<()> {
+        let Some(config_client) = self.context_config(&context_id)? else {
+            bail!("context '{}' does not exist", context_id);
+        };
+
+        let external_client = self.external_client(&context_id, &config_client)?;
+
+        external_client
+            .config()
+            .grant(&granter_id, &[(grantee_id, capability)])
+            .await
+    }
+
+    pub async fn revoke_permission(
+        &self,
+        context_id: ContextId,
+        revoker_id: PublicKey,
+        revokee_id: PublicKey,
+        capability: Capability,
+    ) -> eyre::Result<()> {
+        let Some(config_client) = self.context_config(&context_id)? else {
+            bail!("context '{}' does not exist", context_id);
+        };
+
+        let external_client = self.external_client(&context_id, &config_client)?;
+
+        external_client
+            .config()
+            .revoke(&revoker_id, &[(revokee_id, capability)])
+            .await
+    }
```

Comment:
so we expect the caller to call these manually

this is already the case [in the server](https://github.com/Nathy-bajo/core/blob/bbb1db5cbe250a3b51b4fcd48e99d282de3398dd/crates/server/src/admin/handlers/context/grant_capabilities.rs#L26-L39), you just need to repro the same in merod's interactive CLI, after which we can remove these two

Reply:
Alright!